Java/Greedy-Algorithm/Job_Sequencing.java

```diff
@@ -0,0 +1,139 @@
+//JOB SEQUENCING IN JAVA USING GREEDY APPROACH
+
+import java.util.*;
+
+class Job_Sequencing {
+
+    // Declaring variables
+    int n; // Size of the array/ No of jobs
+    String a[]; // String array to store job numbers
+    int b[]; // Integer array to store the profit associated with each job
+    int c[]; // Integer array to store the deadline associated with each job
+
+    // Input function to take input from user
+    void input() {
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Enter nnumber of jobs: ");
+        n = sc.nextInt();
+        a = new String[n];
+        b = new int[n];
+        c = new int[n];
+        System.out.println("Enter Job No, Profit and Deadline of the job");
+        for (int i = 0; i < n; i++) {
+            a[i] = sc.next();
+            b[i] = sc.nextInt();
+            c[i] = sc.nextInt();
+        }
+        sc.close();
+
+        System.out.println("Arranged Order of Jobs: ");
+        System.out.print("Jobs:     ");
+        for (int i = 0; i < n; i++) {
+            System.out.print(a[i] + " ");
+        }
+        System.out.println();
+
+        System.out.print("Profit:    ");
+        for (int i = 0; i < n; i++) {
+            System.out.print(b[i] + " ");
+        }
+        System.out.println();
+
+        System.out.print("Deadline:   ");
+        for (int i = 0; i < n; i++) {
+            System.out.print(c[i] + " ");
+        }
+        System.out.println();
+    }
+
+    // Arranging jobs in descending order if not sorted
+    void sort() {
+        int temp;
+        String temp1;
+        for (int i = 0; i < n - 1; i++) {
+            for (int j = i + 1; j < n; j++) {
+                if (b[i] < b[j]) {
+                    temp = b[i];
```

Comment:
create a swap function instead of writing same code again and again

Reply:
@Rancho2002 creating a swap function is giving me errors. So, can I keep it the same?

Reply:
okay then its all right.